sktime/registry/_base_classes.py

```diff
@@ -77,6 +81,16 @@
     ("regressor", BaseRegressor, "time series regressor"),
     ("forecaster", BaseForecaster, "forecaster"),
     ("transformer", BaseTransformer, "time series transformer"),
+    (
+        "transformer-pairwise-tabular",
```

Comment:
Minor comment:
Inconsistency in tag and name of Base Class. Tag has tabular at the end, but BaseClass does not